exercises/practice/eliuds-eggs/.meta/example.erl

```diff
@@ -0,0 +1,10 @@
+-module(example).
+
+-export([eggCount/1]).
+
+
+eggCount(Number) -> eggCount(Number, 0).
+
+eggCount(0, Acc) -> Acc;
+eggCount(Number, Acc) when Number > 0 ->
+    eggCount(Number bsr 1, Acc + (Number band 1)).
```

Comment:
```suggestion
-export([egg_count/1]).


egg_count(Number) -> egg_count(Number, 0).

egg_count(0, Acc) -> Acc;
egg_count(Number, Acc) when Number > 0 ->
    egg_count(Number bsr 1, Acc + (Number band 1)).
```

Please stay consistent with the naming conventions in the repo. Unless I missed some exercises, snake case is preferred for function names.